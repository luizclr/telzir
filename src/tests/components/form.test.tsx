import React from "react";
import Form from "../../components/form";
import { shallow } from "enzyme";

describe("<Form />", () => {
  it("should render", () => {
    const wrapper = shallow(<Form />);

    expect(wrapper).toMatchSnapshot();
  });
});
