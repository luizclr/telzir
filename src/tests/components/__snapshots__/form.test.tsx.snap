// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> should render 1`] = `
<form
  onSubmit={[Function]}
>
  <div
    className="row"
  >
    <div
      className="form-group"
    >
      <label>
        From
      </label>
      <select
        name="originCode"
        onChange={[Function]}
      >
        <option
          disabled={true}
          selected={true}
        />
        <option
          key="11"
        >
          11
        </option>
        <option
          key="16"
        >
          16
        </option>
        <option
          key="17"
        >
          17
        </option>
        <option
          key="18"
        >
          18
        </option>
      </select>
    </div>
    <div
      className="form-group"
    >
      <label>
        To
      </label>
      <select
        name="destinyCode"
        onChange={[Function]}
      >
        <option
          disabled={true}
          selected={true}
        />
        <option
          key="11"
        >
          11
        </option>
        <option
          key="16"
        >
          16
        </option>
        <option
          key="17"
        >
          17
        </option>
        <option
          key="18"
        >
          18
        </option>
      </select>
    </div>
    <div
      className="form-group"
    >
      <label>
        Minutes
      </label>
      <input
        name="minutes"
        onChange={[Function]}
        type="number"
      />
    </div>
  </div>
  <div>
    <label>
      Plan
    </label>
    <select
      name="plan"
      onChange={[Function]}
    >
      <option
        disabled={true}
        selected={true}
      />
      <option
        key="30"
      >
        30
      </option>
      <option
        key="60"
      >
        60
      </option>
      <option
        key="120"
      >
        120
      </option>
    </select>
  </div>
  <div
    className="py-10"
  >
    <button
      className="button button__fill--green disabled"
      type="submit"
    >
      calculate
    </button>
  </div>
</form>
`;
